docs/guides/sdks/android-sdk/send-consent-dma-android.md

```diff
@@ -80,72 +80,39 @@ public class MainActivity extends AppCompatActivity {
 
 ## Manually collect consent data
 
-If your app does not use a CMP compatible with TCF v2.2, utilize the SDK API detailed below to provide the consent data directly to the SDK.
-
-1. [Initialize the SDK](https://dev.appsflyer.com/hc/docs/android-sdk-reference-appsflyerlib#init) from the Application class. 
-2. In the `Activity` class, determine whether the GDPR applies or not to the user. 
-
-### When GDPR applies to the user
-
-If GDPR applies to the user, perform the following: 
+To manually collect consent data, perform the following:
+
+1. [Initialize the SDK](https://dev.appsflyer.com/hc/docs/android-sdk-reference-appsflyerlib#init) from the Application class.
+2. In the `Activity` class, determine whether the GDPR applies or not to the user.
+3. Determine whether the consent data is already stored for this session.
+    - If there is no consent data stored, show the consent dialog to capture the user consent decision.
+    - If there is consent data stored continue to the next step.
+4. To transfer the consent data to the SDK, create an object called [`AppsFlyerConsent`](https://dev.appsflyer.com/hc/docs/android-sdk-reference-appsflyerconsent)  with the following optional parameters:
+    - `isUserSubjectToGDPR` - Indicates whether GDPR applies to the user.
+    - `hasConsentForDataUsage` - Indicates whether the user has consented to use their data for advertising purposes.
+    - `hasConsentForAdsPersonalization` - Indicates whether the user has consented to use their data for personalized advertising purposes.
+    - `hasConsentForAdStorage` - indicates whether the user has consented to store or access information on a device.
+  
 
-1. Given that GDPR is applicable to the user, determine whether the consent data is already stored for this session.
-   1. If there is no consent data stored, show the consent dialog to capture the user consent decision. 
-   2. If there is consent data stored continue to the next step. 
-2. To transfer the consent data to the SDK create an object called `AppsFlyerConsent` using the `forGDPRUser()` method with the following parameters:
-   - `hasConsentForDataUsage` - Indicates whether the user has consented to use their data for advertising purposes.
-   - `hasConsentForAdsPersonalization` - Indicates whether the user has consented to use their data for personalized advertising purposes.
-3. Call `setConsentData()` with the `AppsFlyerConsent` object.
-4. Call `start()`.   
+> 📘 Note
+>   
+> The SDK will omit any parameters from the event that are not provided by the app.
```

Comment:
I would rephrase:
SDK registers only parameters which are explicitly passed to this method